.github/workflows/greetings.yml

```diff
@@ -0,0 +1,21 @@
+name: Greetings
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: anjali1102/Greeting-action@main
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          issue_message: 'Hello @${{github.actor}}, <br>
+            Thank you for opening an issue. :partying_face: <br>
+            To get assigned to this particular issue please use <b>/assign</b><br>
+            <i> Check this <a href="https://docs.google.com/document/d/1WYYc2ymImxS6-TKB6pRX77A9zpXb_ujGjdpfMaTDXwc/edit">guide</a> before contributing.'
```

Comment:
Don't copy-paste actions from other projects (In this case NeoAlgo) to other projects. It kills the open-source spirit.